src/backends/postgresql/session.cpp

```diff
@@ -74,6 +76,13 @@ void postgresql_session_backend::connect(
                          : "SET extra_float_digits = 2",
         "Cannot set extra_float_digits parameter");
 
+    {
+        std::ostringstream oss;
+        oss << "SET staetment_timeout = " << statement_timeout_;
```

Comment:
Misprint: `staetment_timeout` instead of `statement_timeout` (and also in line 83)

Reply:
@Mingun Thank you for the review. AT, Having no local dev environment for SOCI, I'm completely relying on the CI builds. I'll fix it.